society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -315,6 +347,9 @@
       <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
         <text variable="title" form="short" font-style="italic" text-case="title"/>
       </else-if>
+      <else-if variable="reviewed-author">
+        <text variable="title" font-style="italic" prefix="review of "/>
```

Comment:
How about this: it uses the `reviewed-title` variable where available and otherwise falls back to your solution. `reviewed-title` isn't currently in Zotero but is available in CSL, will likely be in Zotero in the future and can currently be used in Zotero using Extra (and in other tools using CSL). 

```suggestion
        <choose>
          <if variable="reviewed-title" match="any">
            <group delimiter=", ">
              <text variable="title" quotes="true"/>
              <text variable="reviewed-title" prefix="review of "/>
            </group>
          </if>
          <else>
            <text variable="title" font-style="italic" prefix="review of "/>
          </else>
        </choose>
```

Reply:
Thanks, that looks like a reasonable suggestion. Two quick thoughts on the implementation:
1) You probably need line 355 "reviewed-title" variable to be in italics
2) You probably need changes near lines 299 & 320 for complete coverage